README.md

```diff
@@ -187,6 +187,67 @@ result = my_zosmf.get_info()
 ```
 The result will be a JSON object containing z/OSMF information
 
+## Local setup for macOs
+
+Make sure that the current python version is `3.10` or older
+
+If it is in an older version , you can downgrade to a lower versions by following the below steps:
```

Comment:
Should this be something like "If your Python version is newer than 3.10, you can downgrade to a lower version by using a tool such as pyenv."

I think since the usage of `pyenv` isn't mandatory here and is separate from these setup steps, it may not apply to all users reading this. In this case, I would recommend mentioning `pyenv` as an option, but without going into detail on how to use it.